docs/data/deployments.yml

```diff
@@ -221,3 +221,12 @@
   dask: dask-jobqueue
   jupyter: Single user notebook servers
   kind: hpc
+  
+- name: JASMIN Cluster-as-a-Service
+  description: Dynamic, user-provisioned Pangeo clusters running on the JASMIN Cloud. Supported by UK Science and Technology Facilities Council.
```

Comment:
@kaedonkers 

We have to be a bit sensitive here politically. Technically it is a NERC facility that is operated by STFC. We should probably capture that.

Reply:
```Dynamic, user-provisioned Pangeo clusters running on the JASMIN Cloud. UK Natural Environment Research Council (NERC) facility, operated by UK Science and Technology Facilities Council (STFC).```?

Reply:
Looks good to me.